dozer/cogs/levels.py

```diff
@@ -317,6 +334,82 @@ async def checkrolelevels(self, ctx):
     `{prefix}checkrolelevels`: Returns an embed of all the role levels 
     """
 
+    @group(invoke_without_command=True, aliases=["moderatelevels", "levelsmoderation"])
+    @guild_only()
+    async def adjustlevels(self, ctx):
+        """Allows for moderators to adjust a members level/xp"""
+        await ctx.send(f"Invalid subcommand\nFor help with adjustlevels use `{ctx.prefix}help adjustlevels`")
+
+    adjustlevels.example_usage = """
+    `{prefix}adjustlevels setlevel <@Snowplow or "Snowplow"> 15`:\n Sets member Snowplow's level to 15 
+    `{prefix}adjustlevels changexp <@Snowplow or "Snowplow"> -1500`:\n Changes member Snowplow's xp by -1500xp 
+    `{prefix}adjustlevels swapxp <@Snowplow or "Snowplow"> <@Dozer or "Dozer">`:\n Swaps Snowplow's xp with Dozer's xp
+    `{prefix}adjustlevels transferxp <@Snowplow or "Snowplow"> <@Dozer or "Dozer">`:\n Adds Snowplow's xp to dozer's xp
+    """
+
+    @adjustlevels.command()
+    @guild_only()
+    @has_permissions(manage_messages=True)
+    async def setlevel(self, ctx, member: discord.Member, level: int):
+        """Changes a members level to requested level"""
+        if level >= 100000:
+            raise BadArgument("Requested level is too high!")
+        entry = await self.load_member(ctx.guild.id, member.id)
+        xp = self.total_xp_for_level(level)
+        DOZER_LOGGER.debug(f"Adjusting xp for user {member.id} to {xp}")
+        entry.total_xp = xp
+        await self.sync_member(ctx.guild.id, member.id)
+        e = discord.Embed(color=blurple)
+        e.add_field(name='Success!', value=f"I set {member}'s level to {level}")
+        e.set_footer(text='Triggered by ' + ctx.author.display_name)
+        await ctx.send(embed=e)
+
+    @adjustlevels.command(aliases=["addxp"])
+    @guild_only()
+    @has_permissions(manage_messages=True)
+    async def changexp(self, ctx, member: discord.Member, xp_amount: int):
+        """Changes a members xp by a certain amount"""
```

Comment:
This description is misleading. Changing XP implies that this command sets the member's XP to that level, while this function actually adds it to the current total XP.